src/components/Node.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <li>
+    <template v-if="hasChildren">
+      <span class="open-toggle" @click.prevent="toggle">
+        <Icon :name="icon" class="fa-xs" fixed-width />
+      </span>
+    </template>
+
+    <span class="node version" v-if="routable">
+      <router-link
+        :to="{ path: 'reader', query: { urn: metadata.firstPassageUrn } }"
+      >
+        {{ metadata.workTitle }}
+      </router-link>
+    </span>
+    <span v-else class="node monospace">
+      <tt>{{ urn }}</tt>
+    </span>
+
+    <ul class="node-tree" v-if="expanded">
```

Comment:
At "smaller" resolutions (screenshot below is at 1400 x 900), the wrapping can cause the chevrons to drift from the URNs / titles:

![image](https://user-images.githubusercontent.com/629062/73663679-a3152500-4663-11ea-9e4f-f7d63a37ea9c.png)

I think we're going to do more library work in the next sprint, so I don't consider it a blocker, but we might want to try testing out layouts at smaller screen sizes as we're building out widgets in the future.

Reply:
Yep I noticed this too on Friday might when I was on the macbook instead of the monitor, and I was going to fix it up today while clearing a few last details. Agreed we should start thinking more responsive though, I'll endeavour to do that from here-out.